examples/stm32/xively_demo_mt/Src/system_stm32f4xx.c

```diff
@@ -1,21 +1,21 @@
 /**
   ******************************************************************************
-  * @file    system_stm32f4xx.c

```

Comment:
was this file update due to an SDK update or similar? so what caused the update of this file and why is it needed, thx.